test/test_helpers/project.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+module TestHelpers
+  module Project
+    include TestHelpers::AppType
+
+    def setup
+      project_context
+      super
+    end
+
+    def project_context(root = File.join(MiniTest::Test::FIXTURE_DIR, 'project'))
```

Comment:
If the root parameter is always prefixed by the fixtures root then maybe just do that inside the method? That way when you call it in the other setup methods you only need to pass `project_context('app_type/rails'))` or something like that? Although then it might break windows tests so it might not be worth it.

Reply:
I like this idea. We don't need to worry about windows paths because we're specifying people should use Linux Subsystem for Windows.